client/src/legacy/GridField.js

```diff
@@ -335,28 +335,42 @@ $.entwine('ss', function($) {
     },
   });
 
-  $('.grid-field .ss-gridfield-item').entwine({
+  $('.grid-field .ss-gridfield-item td:not(.action-menu)').entwine({
+    onmatch: function() {
+      this._super();
+      // Cache data on whether the parent row has interactive elements and any link to a form
+      var formLink = this.closest('tr').find('.edit-link, .view-link').prop('href');
+      var cssSelector = [
+        'a[href]',
+        'button:not([disabled])',
+        'input:not([disabled]):not([type="hidden"])',
+        'select:not([disabled])',
+        'textarea:not([disabled])',
+        '[tabindex]:not([tabindex="-1"])',
```

Comment:
```suggestion
        'a[href]',
        ':input:not(:disabled, :hidden)',
        '[tabindex]:not([tabindex="-1"])',
```
See jQuery docs:
- https://api.jquery.com/input-selector/
- https://api.jquery.com/disabled-selector/
- https://api.jquery.com/hidden-selector/